src/styles/components/molecules/Footer.scss

```
@@ -0,0 +1,18 @@
+.footer-social {
+  text-align: center;
+  color: $color-primary-white;
+  p {
+    margin-bottom: $space-elements-small;
+    font-family: $font-primary;
+    font-weight: $font-weight-bold;
+  }
+  .social {
+    font-size: $font-size-p;
+    & svg {
+      margin-right: 8px;
```

Try to use the spaces variables for give margin to an element.
Here can be `$space-elements-small`
```suggestion
      margin-right: $space-elements-small;
```

---

Ready